packages/@react-spectrum/s2/src/TableView.tsx

```diff
@@ -410,6 +413,8 @@ export function TableBody<T extends object>(props: TableBodyProps<T>) {
 
   return (
     <RACTableBody
+      // @ts-ignore
+      ref={domRef}
```

Comment:
Note these `ts-ignore`. At the moment the RAC Table subcomponents only accept their refs of type HTMLTableSectionElement/etc even though those elements turn into divs when virtualized. I'm currently trying to finagle how we can make those elements accept refs of both their table element type as well as HTMLDiv but for now just skipping lint for those